www/templates/diary/list.html

```diff
@@ -40,73 +40,11 @@
         -->
 
 		<div collection-repeat="tripgj in data.currDayTripWrappers" item-height="tripgj.common.different? itemHt + 80 : itemHt + 30">
-           <div class="start-time-tag">{{tripgj.display_start_time}}</div>
-            <div  style="padding-left: 19%;">
-
```

Comment:
Ignoring this file in the review as well since it is obsolete.

Reply:
_**Confirmed this change as complete**_